src/control/server/storage/scm/ipmctl.go

```diff
@@ -460,14 +463,15 @@ func (cr *cmdRunner) prepReset(req storage.ScmPrepareRequest, scanRes *storage.S
 
 	for _, dev := range scanRes.Namespaces {
 		if err := cr.removeNamespace(dev.Name); err != nil {
-			return nil, err
+			cr.log.Noticef("removeNamespace: %s", err.Error())
+			// Continue to reset regions.
```

Comment:
I'm curious about what happens practically if it's not possible to remove the namespaces for some reason. If there are undeleted namespaces on the DIMM, will `ipmctl create -goal` still work? If not, it would be better to return the error here.

Do we know why namespaces might not be deleted? All I can think of is the device being busy.

Reply:
yes the force create -goal should purge namespaces on the NVDIMM on reboot
yes device busy could cause refusal to delete but they will still be removed on reboot

Reply:
updated